exercises/05-reselect/src/modules/root/components/header.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import { getTitle } from 'modules/users/users-selectors'
+
+export const Header: React.FC = () => {
```

Comment:
I'd say `React.FC` is unnecessary

Reply:
Some would say, React.FC is an antipattern - https://github.com/facebook/create-react-app/pull/8177
I do not have strong opinion

Reply:
Interesting link, I was not aware of that.

In one of the previous [PR](https://github.com/salsita/react-training/pull/44)s we decided to keep `React.FC` because of
- implicit return type
- `React.FC<Props>` was used for typing of components with props, so an empty variant remained for consistency 

I'm unsure now, whether to reopen this discussion again. I guess we could create a separate issue and if the decision falls to get rid of `React.FC`, it'll be done in all exercises at once.

Reply:
I've created a new issue for this discussion: https://github.com/salsita/react-training/issues/61